### PR TITLE
Feature/#855/header refactor

### PR DESCRIPTION
**Pull request by MiltonTulli**
# Description
Header refactor
Closes https://github.com/DXgovernance/dxvote/issues/855

### CODE REVIEW

File: src/locales/en/translation.json [hunk 1]
```diff
@@ -4,9 +4,16 @@
     "addOption": "Add Option",
     "editOption": "Edit Option",
     "deleteOption": "Delete Option",
-    "optionLabel": "Option Label"
+    "optionLabel": "Option Label",
```
Comment (rossneilson) [inline src/locales/en/translation.json]: Not just something for this PR but we could quickly fix it here. I don't think we should be putting non guild specific text in guilds object, most of these will be reused outside of guilds

Comment (MiltonTulli) [reply src/locales/en/translation.json]: Agree. Not sure what is generic and what is guild specific. Hard one. Should I put all in the root obj? I think the only specific one for guilds is `create guild`

Comment (rossneilson) [reply src/locales/en/translation.json]: Yeah, safe bet to go for anything that says guild in it for now

Comment (MiltonTulli) [reply src/locales/en/translation.json]: Sure. Changed translation file and refactor. I think we can refactor later if we find some specific guild stuf, but for now almost all sounds like generic.

**Review by rossneilson — APPROVED** [review]

**Pull request merged**